### PR TITLE
Add a note about ports to open for docker setup

### PR DESCRIPTION
**Pull request by nolim1t**
I've been trying to get the docker version to work with the documentation, however the port is not `3000` as documented but it is `3300`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,6 +100,8 @@ If you have a permanent public IP on your internet connection and you want your
 
 If you plan to use your Sphinx clients within the local network, then you do not have to do anything special.
 
+If you are using the Docker file, the port may be 3300 ([1](https://github.com/stakwork/sphinx-relay/blob/master/Dockerfile#L32))
```
Comment (dennisreimann) [inline README.md:103]:
```suggestion
If you are using the Docker file, the port is `3300` ([1](https://github.com/stakwork/sphinx-relay/blob/master/Dockerfile#L32))
```

Comment (dennisreimann) [reply README.md:103]: I think it's clearer this way.

**Comment by nolim1t** [comment]
Thanks for the feedback

**Comment by dennisreimann** [comment]
@Evanfeenstra I think this should get merged – it's valuable information.

**Review by kevkevinpal — APPROVED** [review]

**Pull request merged**